newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/ExternalEventsAutoComplete.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import { string } from 'prop-types';
```

Comment:
Remove this line it's useless

Reply:
I'm gonna do it

Reply:
@Oxey405 you still need to do this :)

Reply:
Yeah I know I'm late and I have school so you know, my calendar is booked up really quickly, I'm gonna do it tomorrow I promise  ! :)

Reply:
@4ian done ! I hope you find no more problems with the PR